quoridorui/quoridor-ui.iml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please try not to commit and push `.iml` files. There are files that define the project structure locally for the IntelliJ IDEA